cellprofiler/modules/convertimagetoobjects.py

```diff
@@ -138,7 +138,7 @@ def display(self, workspace, figure):
             y=0
         )
 
-        figure.subplot_imshow(
+        figure.subplot_imshow_labels(
```

Comment:
We should (eventually) consider converging to one common name for this type of data. In addition to labels, we also use masks and segmented. 🤷‍♀️